recipes/pffft/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.28.0"
+
+
+class ConanPffft(ConanFile):
+    name = "pffft"
+    description = "PFFFT, a pretty fast Fourier Transform."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://bitbucket.org/jpommier/pffft/src/master/"
+    topics = ("fft", "pffft")
+    license = "BSD-like (FFTPACK license)"
+    generators = "cmake"
+    settings = "os", "compiler", "arch", "build_type"
+    exports_sources = ["CMakeLists.txt"]
+    options = {"disable_simd": [True, False], "fPIC": [True, False]}
+    default_options = {"disable_simd": False, "fPIC": True}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        url = self.conan_data["sources"][self.version]["url"]
+        archive_name = os.path.basename(url)
+        archive_name = "jpommier-pffft-" + os.path.splitext(archive_name)[0]
+        os.rename(archive_name, self._source_subfolder)
```

Comment:
This is the boilerplate we usually go with but it requires an extra import

```suggestion
        extracted_dir = glob.glob("jpommier-pffft-*")[0]
        os.rename(extracted_dir, self._source_subfolder)
```

Reply:
In all honesty, I asked on slack what was the recipe that looked the most like what I needed, @SSE4 told me it was sqlite3, and I pretty much copy pasted it. But thanks, next time I'll do better!